### PR TITLE
Melodic devel veiko

### PR DESCRIPTION
**Pull request by Veix123**
Since the driver package was updated, we can now go live with this one. CI in place, and builds are passing. Please @RValner and @kruusamae have look if you find anything that need improvements.

### CODE REVIEW

**Review by RValner — APPROVED** [review]
Looks good.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+roslaunch robotont_support robotont_bringup.launch
+```
+This command will start robotont driver along with the realsense 3D camera driver, loads robot description to the parameter server, and starts state publishers.
+
+
+* If you do not need the realsense camera, you can save the battery by running:
+
+```bash
+roslaunch robotont_support robotont_bringup.launch realsense:=false
+```
```
Comment (kruusamae) [inline README.md]: we should highlight that robotont_bringup.launch **needs to be launched on the Robotont on-board computer**.

**Review by kruusamae — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Veix123) [reply README.md]: > we should highlight that robotont_bringup.launch **needs to be launched on the Robotont on-board computer**.

can you have another look @kruusamae?

Comment (kruusamae) [reply README.md]: yes, looks good now.

**Review by kruusamae — APPROVED** [review]

**Pull request merged**